apptentive/src/com/apptentive/android/sdk/module/engagement/interaction/model/NavigateToLinkInteraction.java

```diff
@@ -26,4 +27,12 @@ public String getUrl() {
 		}
 		return null;
 	}
+
+	public boolean isNewTask() {
+		InteractionConfiguration configuration = getConfiguration();
+		if (configuration != null && !configuration.isNull(KEY_NEW_TASK)) {
+			return configuration.optBoolean(KEY_NEW_TASK, true);
```

Comment:
Default is `true` for opening in a new task. What does this mean for opening a link into your own app? Does that fail, open in a "new" task, or open in the same task?